integration/airflow/tests/integration/gcloud/gcloud-service-key.json

```diff
@@ -0,0 +1 @@
+
```

Comment:
Is the `gcloud-service-key.json` file needed for the testing?

Reply:
Removed.